README.md

```diff
@@ -121,6 +121,17 @@ Type "help", "copyright", "credits" or "license" for more information.
 
 You should see a plot!
 
+### image mimetype support
+
+If you want to display a wider variety of objects using ipython's mimetype capturing, load our bundled ipython extension:
+
+```ipython
+$ %load_ext itermplot.ipython
```

Comment:
If you're okay with adding `load_ipython_extension` to `itermplot/__init__.py`, it'd make this a cleaner `%load_ext itermplot`